cmd/zdb/zdb.c

```diff
@@ -4280,6 +4554,25 @@ zdb_count_block(zdb_cb_t *zcb, zilog_t *zilog, const blkptr_t *bp,
 		    [BPE_GET_PSIZE(bp)]++;
 		return;
 	}
+	{
+		int bin = highbit64(BP_GET_PSIZE(bp))-1;
```

Comment:
nit: `highbit64(BP_GET_PSIZE(bp))-1;` -> `highbit64(BP_GET_PSIZE(bp)) - 1;`, here and on 4564 and 4570